concepticondata/conceptlists/Smeltzer-1995-300.tsv

```diff
@@ -0,0 +1,301 @@
+Smeltzer-1995-300-247	247	côté droit	1019	RIGHT	right side
+Smeltzer-1995-300-248	248	côté gauche	244	LEFT	left side
+Smeltzer-1995-300-249	249	peu de	1242	FEW	few
+Smeltzer-1995-300-250	250	autre	197	OTHER	other, another
+Smeltzer-1995-300-251	251	où?	1237	WHERE	where
+Smeltzer-1995-300-252	252	ici	136	HERE	here
+Smeltzer-1995-300-253	253	là-bas	3411	OVER THERE	over there
+Smeltzer-1995-300-254	254	près de	1942	NEAR	near
+Smeltzer-1995-300-255	255	loin	1406	FAR	far
+Smeltzer-1995-300-256	256	comment?	1239	HOW	how
+Smeltzer-1995-300-257	257	quand?	1238	WHEN	when
+Smeltzer-1995-300-258	258	aujourd'hui	1283	TODAY	today
+Smeltzer-1995-300-259	259	hier	1174	YESTERDAY	yesterday
+Smeltzer-1995-300-260	260	demain	1329	TOMORROW	tomorrow
+Smeltzer-1995-300-261	261	vivant	1422	BE ALIVE	alive
+Smeltzer-1995-300-262	262	sale	1230	DIRTY	dirty
+Smeltzer-1995-300-263	263	mûr	178	RIPE	ripe
+Smeltzer-1995-300-264	264	tranchant (aigu)	1396	SHARP	sharp
+Smeltzer-1995-300-265	265	mouillé	1726	WET	wet
+Smeltzer-1995-300-266	266	pourri	1728	ROTTEN	rotten
+Smeltzer-1995-300-267	267	chasser le gibier	1435	HUNT	to hunt game
+Smeltzer-1995-300-268	268	respirer	1407	BREATHE	to breathe
+Smeltzer-1995-300-269	269	bâiller	1028	YAWN	to yawn
+Smeltzer-1995-300-270	270	danser	1879	DANCE	to dance
+Smeltzer-1995-300-271	271	pleurer	1839	CRY	to cry
+Smeltzer-1995-300-272	272	interroger	1309	ASK (INQUIRE)	to ask
+Smeltzer-1995-300-273	273	donner (ordre)	1998	COMMAND	to give a command
+Smeltzer-1995-300-274	274	écrire	1672	WRITE	to write
+Smeltzer-1995-300-275	275	penser / réfléchir	1415	THINK (REFLECT)	to think / reflect
+Smeltzer-1995-300-276	276	souffrir	1101	SUFFER	to suffer
+Smeltzer-1995-300-277	277	aimer	923	LOVE	to love
+Smeltzer-1995-300-278	278	chercher	1468	LOOK FOR	to search
+Smeltzer-1995-300-279	279	trouver	719	FIND	to find
+Smeltzer-1995-300-280	280	prendre	1749	TAKE	to take
+Smeltzer-1995-300-281	281	préparer	2909	PREPARE	to prepare
+Smeltzer-1995-300-282	282	cacher	602	HIDE (CONCEAL)	to hide (something)
+Smeltzer-1995-300-283	283	mentir	601	LIE (MISLEAD)	to lie
+Smeltzer-1995-300-284	284	arriver	1387	ARRIVE	to arrive
+Smeltzer-1995-300-285	285	suivre	1085	FOLLOW	to follow
+Smeltzer-1995-300-286	286	tresser	3294	BRAID (VERB) OR WEAVE (BASKET)	to braid, weave
+Smeltzer-1995-300-287	287	courir	1519	RUN	to run
+Smeltzer-1995-300-288	288	tourner	1588	TURN	to turn
+Smeltzer-1995-300-289	289	tirer (arme)	1172	SHOOT	to shoot
+Smeltzer-1995-300-290	290	essuyer	1454	WIPE	to wipe
+Smeltzer-1995-300-291	291	balayer	230	SWEEP	to sweep
+Smeltzer-1995-300-292	292	ouvrir	1156	OPEN	to open
+Smeltzer-1995-300-293	293	casser	2558	BREAK (CLEAVE)	to break
+Smeltzer-1995-300-294	294	couper	1432	CUT	to cut
+Smeltzer-1995-300-295	295	percer	398	PIERCE	to pierce
+Smeltzer-1995-300-296	296	lutter	1423	FIGHT	to fight
+Smeltzer-1995-300-297	297	nouer (attacher)	1917	TIE	to tie
+Smeltzer-1995-300-298	298	délier	352	UNTIE	to untie
+Smeltzer-1995-300-299	299	battir			to hit, beat?
```

Comment:
There's currently an issue open on HIT etc, but based on the discussion I would say:
Is it to hit intentionally a single time? Then it's STRIKE. If intentional several times, POUND.

Reply:
In this case, it doesn't matter because this word is very likely to be mistyped and my translation given here corresponds to the word 'battre' that in its turn means to hit or beat. The idea was to leave such ambiguous glosses unmapped just to be on the safe side.